packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/BankPicker/template.success.tsx

```diff
@@ -81,43 +87,44 @@ const Success: React.FC<Props> = props => {
                 <Icon name='bank-filled' color='blue600' size='16px' />
               </DisplayPaymentIcon>
             </Col>
-            <Col>
+            <Col style={{ width: '100%' }}>
               <Content>
                 <Value asTitle>{beneficiary.name}</Value>
                 <Title asValue>{beneficiary.agent.account}</Title>
               </Content>
             </Col>
+            {props.beneficiary?.id === beneficiary.id ? (
+              <Icon
+                name='checkmark-circle-filled'
+                size='24px'
+                color='green600'
+                role='button'
+                style={{ justifyContent: 'flex-start' }}
+              />
+            ) : (
+              <Image
+                name='circle-empty'
+                width='24px'
+                height='24px'
+                style={{ justifyContent: 'flex-start' }}
+              />
+            )}
```

Comment:
we have this code on 2 places, it's not a mandatory but would be cool to make it as one helper function or service and reuse it on both places

Reply:
@milan-bc moved it to a service